tasks/utils.js

```diff
@@ -79,9 +78,21 @@ const contractNetworks = {
 
 const getPartnerKey = (index) => `partner${index}`;
 
+const defaultRevenueSharingAddressesPath = '../revenue-sharing-addresses.json';
+let revenueSharingAddresses = undefined;
+const getRevenueSharingAddressesFromFile = (
+    filename = defaultRevenueSharingAddressesPath
+) => {
+    if (revenueSharingAddresses === undefined) {
```

Comment:
I think this line could be like this:
`if(!revenueSharingAddresses) {`

Reply:
Although the result may be the same, I usually prefer to be explicit and check for the type as well.
In this specific case, we set the initial value to `undefined` and this is the only case in which I would read it from the file.